core/src/test/java/org/mule/tck/junit4/AbstractMuleTestCase.java

```diff
@@ -55,6 +55,11 @@
     public static final String TEST_TIMEOUT_SYSTEM_PROPERTY = "mule.test.timeoutSecs";
 
     /**
+     * Default timeout for waiting for responses
+     */
+    public static final int RECEIVE_TIMEOUT = 5000;
```

Comment:
AbstractMuleTestCase is supposed to be for really unit tests (the small ones), so a timeout of 5 seconds is not appropriate